docs/_api-readme.md

```diff
@@ -0,0 +1,27 @@
+## Main exports
+Import the main lit-html functions from the `lit-html` module:
+
+```js
+import {render, html, svg} from 'lit-html';
+```
+
+- [html](./globals.html#html)
+- [svg](./globals.html#svg)
+- [render](./globals.html#render)
+
+## Directives
+Import from individual modules in the `lit-html/directives/` folder:
+
+```js
+import {repeat} from 'lit-html/directives/repeat.js';
+```
+
+- [asyncAppend](./globals.html#asyncAppend)
```

Comment:
Not sure why, but the actual anchors TypeDoc is generating are lowercase, so these links are broken for camelCase directives (asyncReplace, classMap) although they work fine for the others (guard, repeat).